lib/components/artist/shows/index.js

```diff
@@ -43,11 +46,9 @@ const styles = StyleSheet.create({
   },
 });
 
-export default Relay.createContainer(Shows, {
-  initialVariables: {
-    artistID: null,
-  },
+var pastShowsFragment = windowDimensions.width > 700 ? LargeShowsList.getFragment('shows') : SmallShowsList.getFragment('shows');
```

Comment:
This could also go inside `createContainer` but it might be more readable up here.